### ISSUE TITLE
Help with Setup

### ISSUE THREAD

**Issue by btsybm** [BerryWorksSoftware/edireader#21]
Any source you can point me to in terms of the Java that will be needed. I operate primarily in C# and JS.

**Comment by canabrook** [comment]
Any Java 11 or later will work fine.  One good choice would be AdoptOpenJDK https://adoptopenjdk.net  
Let me know if you need more info.

**Comment by Abilasha28** [comment]
Will it work for Java 8 version?

**Comment by canabrook** [comment]
Yes. All of the modules and components were developed with Java 8 and earlier, and are runnable on Java JDKs 8 and later (11, 14, …). A git branch is contained to syntax from Java 8, while the master branch uses a few Java 11 syntax features. The intent is to remain runnable in Java 8 applications for the foreseeable future.

Scott
***@***.***
